packages/core/lib/testing/Test.js

```diff
@@ -244,7 +244,7 @@ const Test = {
     const migrateConfig = config.with({
       reset: true,
       resolver: resolver,
-      quiet: true
+      quiet: false
```

Comment:
Should we set this to `false` if-and-only-if `--migrate-none` is turned on? `truffle test` silences migrate output on purpose right now, and I don't think we want to change that most of the time.

I do like your use of this mechanism, though! Let the migrate system do its own logging!

Reply:
Yeah I'm also confused by this; I don't think we should be setting it to `false` at all?  Presumably we want to suppress the printout regardless.

Reply:
will change it to false if and only if --migrate-none